.github/workflows/ci.yml

```diff
@@ -12,33 +12,6 @@ env:
   GOPROXY: https://proxy.golang.org/
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    timeout-minutes: 25
-    steps:
-      -
-        name: Checkout
-        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # https://github.com/actions/checkout/releases/tag/v3.3.0
-      -
-        name: Unshallow
-        run: git fetch --prune --unshallow
-      -
-        name: Set up Go
-        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # https://github.com/actions/setup-go/releases/tag/v3.5.0
-        with:
-          go-version-file: ".go-version"
-      -
-        name: Generate provider schemas
-        run: |
-          go generate ./internal/schemas
-          du -h -s ./internal/schemas/data
-      -
-        name: Snapshot build (cross-platform)
-        uses: goreleaser/goreleaser-action@8f67e590f2d095516493f017008adc464e63adb1 # https://github.com/goreleaser/goreleaser-action/releases/tag/v4.1.0
-        with:
-          version: latest
-          args: build --snapshot --skip-post-hooks
-
   test:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 10
```

Comment:
I don't expect https://github.com/hashicorp/terraform-ls/issues/1085 to be addressed in this PR/context here, but just asking for our future selves: Can you confirm we'll still have some way of reading the artifacts (e.g. using the `actions/download-artifacts`) even when it's a separate workflow, rather than just a job?

Reply:
What artifacts are you referring? For example, if we want to run the tests with schemas present, we would also need to generate those in this workflow.

Reply:
In that case - and in relation to my other comment - would it make sense to only extract the actual building to `build-pr` workflow and keep the schema generation here? By artifacts I mean the provider schema JSON files.